src/communication/command/start.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+##
+## EPITECH PROJECT, 2024
+## Gomoku
+## File description:
+## start
+##
+
+from src.Map.map import *
+
+def handle_start(size:int, map:Map) -> None:
```

Comment:
Please keep a spae between the var and the type declaration // same for all the other declarations.